packages/google_maps_flutter/google_maps_flutter_android/lib/src/utils/cluster_manager.dart

```diff
@@ -0,0 +1,34 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:google_maps_flutter_platform_interface/google_maps_flutter_platform_interface.dart';
+
+/// Converts a Set of Cluster Managers into object serializable in JSON.
+Object serializeClusterManagerSet(Set<ClusterManager> clusterManagers) {
+  return clusterManagers
+      .map<Object>((ClusterManager cm) => serializeClusterManager(cm))
+      .toList();
+}
```

Comment:
```suggestion
/// Converts a Set of Cluster Managers into object serializable in JSON.
List<Object> serializeClusterManagerSet(Set<ClusterManager> clusterManagers) {
  return clusterManagers
      .map<Object>((ClusterManager cm) => serializeClusterManager(cm))
      .toList();
}
```

Should the return type here be `List<Object>`?